org/views.py

```diff
@@ -93,3 +93,41 @@ def AddVolunteer(request,org_id):
             return Response({"message":"You are added as a volunteer"},status.HTTP_201_CREATED)
     else:
         return Response({"detail":"Organization not present"},status.HTTP_400_BAD_REQUEST)
+
+@swagger_auto_schema(
+    operation_id="edit_org",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='PUT',
+    request_body = EditOrgSerializer,
+    responses={
+        '200': set_example(responses.update_org_200),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '403': set_example(responses.admin_access_403),
+    }
+)
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def EditOrg(request,org_id):
+    org_count = Org.objects.filter(pk=org_id).count()
```

Comment:
Why are we using `count()` and `filter()` here.
Better we can use `try and catch` exception, with `get()` Method.
```
try:
    org = Org.objects.get(pk=id)
except Org.DoesNotExist:
    org = None
```